data-node/src/main/java/org/graylog/datanode/process/WatchdogWithProcessInfo.java

```diff
@@ -43,6 +43,6 @@ public ProcessInformation processInfo() {
     }
 
     private static ProcessInformation processInfo(Process p) {
-        return new ProcessInformation(p.pid(), p.isAlive(), p.info().startInstant().orElse(null));
+        return ProcessInformation.create(p);
     }
```

Comment:
You can now throw this method away, completely, and reference new `create` method in `map` method in line 41.